docs/processors/standalone/index.mdx

```diff
@@ -4,8 +4,92 @@ sidebar_position: 3
 
 # Standalone processors
 
-:::warning
+These are processors you'd write yourself in cases the [Built-in](/docs/processors/builtin/) ones don't meet your needs. 
+
+## How to write one
+
+Thanks to our [Web Assembly (Wasm) processor](/docs/processors/standalone/how-it-works) you can start writing processors in any language that can be compiled to Web Assembly. As a start, Conduit already provides a [conduit-processor-sdk](https://github.com/ConduitIO/conduit-processor-sdk) that will let you [write a processor](/docs/processors/standalone/building) in Golang.
```

Comment:
`/docs/processors/standalone/building` will be added via https://github.com/ConduitIO/conduit-site/pull/53.